### PR TITLE
Added section to code reviews

### PR DESCRIPTION
**Pull request by alinakazzaa**
- added topic code review workflow
    - planning
    - process

### CODE REVIEW

**Comment by harrythomas1998** [comment]
This looks fine to me

File: README.md [hunk 1]
```diff
@@ -130,3 +130,48 @@ Overly complex structures and unnecessary code can be avoided by effective refac
...
+#### Plan the review
+
+* Identify the goal of the review - functionality, code quality, etc.
+* Focus on single functionality/ aspects of code to maintain focus
+
+Know what you are looking for in the code - what should be included/used and what should be avoided
+
+#### Review process
+
+The goal of a review is to
```
Comment (chriskavanagh97) [inline README.md]: Each time you lead with a point followed by bullet points you should add " : " to keep consistency, as above you have already done this.

For eg:
The goal of a review is to:

File: README.md [hunk 2]
```diff
@@ -130,3 +130,48 @@ Overly complex structures and unnecessary code can be avoided by effective refac
...
+
+* Ensure core functionality is delivered
+* Identify as many bugs as possible before launching the code to production
+
+
+ To keep reviews efficient processes must be
```
Comment (chriskavanagh97) [inline README.md]: same as the previous comment add " : " to keep consistency

**Review by chriskavanagh97 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (alinakazzaa) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (alinakazzaa) [reply README.md]: done

**Pull request merged**